src/shared/components/unsupportedBrowserModal/UnsupportedBrowserModal.tsx

```diff
@@ -14,24 +14,21 @@ export default class UnsupportedBrowserModal extends React.Component<{}, IBrowse
     constructor() {
         super();
 
-        let state;
         if (!browser)  {
-            state = {
+            this.state = {
                 name: "Unsupported",
```

Comment:
seems like at line 64 we are testing a lowercased "unsupported"    is that right?